x-pack/test_serverless/api_integration/test_suites/search/config.ts

```diff
@@ -21,5 +22,22 @@ export default createTestConfig({
   kbnServerArgs: [
     // useful for testing (also enabled in MKI QA)
     '--coreApp.allowDynamicConfigOverrides=true',
+    '--xpack.dataUsage.enabled=true',
```

Comment:
we need to enable this by default to run the tests in MKI.  once we enable this by default in serverless, we can remove this.